packages/pn-commons/src/utility/multilanguage.utility.ts

```diff
@@ -4,6 +4,11 @@ import { LangCode } from '@pagopa/mui-italia';
 
 export const LANGUAGE_SESSION_KEY = 'lang';
 
+export const getSessionLanguage = (): string => {
```

Comment:
Are you sure we get the language in this way? Should we use i18n?

Reply:
Yes because on `pn-commons` we don't have `i18n`

Reply:
Can we pass the language from external?